src/dune_rules/cram/cram_exec.ml

```diff
@@ -443,14 +443,19 @@ let run_and_produce_output ~src ~env ~dir:cwd ~script ~dst ~timeout =
   let lexbuf = Lexbuf.from_string script_contents ~fname:(Path.to_string script) in
   let temp_dir = make_temp_dir ~script in
   let cram_stanzas = cram_stanzas lexbuf in
-  Path.unlink_exn script;
+  (* We don't want the ".cram.run.t" dir around when executing the script. *)
+  Path.rm_rf (Path.parent_exn script);
```

Comment:
Shouldn't we be clearing this directory rather than deleting it? This way we don't have to do the mkdir_p?

Reply:
A strange behaviour is that this directory doesn't appear for directory cram tests, so perhaps we are misconfiguring a directory somewhere. I couldn't find it, so this is the solution I chose.